src/Dangl.SchneidControl/Services/DataLoggingScheduler.cs

```diff
@@ -17,12 +22,38 @@ public Task StartAsync(CancellationToken cancellationToken)
             Task.Run(async () =>
             {
                 DateTime lastStart;
+                DateTime? lastEmailSent = null;
                 while (_isRunning)
                 {
                     lastStart = DateTime.UtcNow;
                     using var scope = _serviceProvider.CreateScope();
                     var dataLoggingService = scope.ServiceProvider.GetRequiredService<IDataLoggingService>();
-                    await dataLoggingService.ReadAndSaveValuesAsync();
+                    var valuesResult = await dataLoggingService.ReadAndSaveValuesAsync();
+                    if (valuesResult.BufferTemperatureTop?.Value > 50 && valuesResult.BufferTemperatureTop?.Unit == "°C" && valuesResult.CurrentHeatingPowerDraw?.Value != 0)
```

Comment:
We should not use the current power to check if the system is online, but rather the `TransferStationStatus` (https://github.com/GeorgDangl/Dangl.SchneidControl/blob/develop/src/Dangl.SchneidControl/Models/Enums/TransferStationStatus.cs#L3). Active should be any status except `OffOrFrostControl` and `Maintenance`.